### DISCUSSION TITLE
Support Plex's new ".plexmatch" metadata format

### DISCUSSION THREAD

**Discussion by flick1999** [pymedusa/Medusa#10462] [Ideas & Feature requests]
Rather than supporting "industry standard" files like tvshow.nfo, Plex has created a new metadata file format to use. See https://support.plex.tv/articles/plexmatch/ for the full details. 

Request: support the new metadata format 

Thank you.

**Comment by p0psicles** [comment]
I'm working on a proof of concept. And I can currently output this as a show's level .plexmatch.
```
Title: Attack on Titan
Year: 2013
tvdbid: 267440
tmdbid: 1429
imdbid: tt0267440
```

Would that be sufficient?
I can't add the episodes, or at least that would require a lot more work. As medusa is currently setup to create for each episode a metadata file. And not for each episode an entry in the .plexmatch.

**Reply by p0psicles** [reply]
I was able to create the following .plexmatch file:
```
Title: Arrow
Year: 2012
tmdbid: 1412
tvdbid: 257655
imdbid: tt0001412
sp: 01: /Season 0/Arrow - S00E01 - Green Arrow & The Canaries - 1080p HDTV.mkv
ep: s08e09: /Season 08/Arrow - S08E09 - Green Arrow & The Canaries - 1080p HDTV.mkv
ep: s08e10: /Season 08/Arrow - S08E10 - Fadeout - 1080p HDTV.mkv
```

Could you test this in plex?

**Reply by flick1999** [reply]
I tested both of these and Plex picked them up as valid.

I agree, it'll be a lot more work to do the episodes. I think getting the simple "this is the show information" one into prod, as in your first example, would be good. That way Plex at least knows what show it's working with.

I can do a separate request for episodes and you can mark it low priority on the roadmap, maybe?

**Reply by flick1999** [reply]
I just tested the dev branch and I see you already worked in episodes. Nicely done!

**Comment by p0psicles** [comment]
Feature merged to develop
